modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/filters/modal_content/selection_filter/SelectionFilter.tsx

```diff
@@ -76,7 +76,7 @@ function Body({
 		return (
 			<ClayAlert displayType="info" title="Info">
 				{Liferay.Language.get(
-					'no-filter-sources-are-available.-create-a-picklist-or-a-vocabulary-for-this-type-of-filter'
+					'no-filter-sources-available.-create-a-picklist-or-a-vocabulary-for-this-type-of-filter'
```

Comment:
Wouldn't be better to update the key in the `Language.properties` file?

There are more sentences like this, x.e  `no-frontend-data-set-filter-client-extensions-are-available.-...`